src/classes/Passage.ts

```diff
@@ -272,4 +273,32 @@ export default class Passage {
             });
         return magicLinkData;
     }
+
+    /**
+   * Get App Info about an app
+   *
+   * @return {Promise<AppObject>} Passage App object
+   */
+    async getApp(): Promise<AppObject> {
+        if (!this.#apiKey) {
+            throw new Error("A Passage API key is needed to make a getUser request");
+        }
```

Comment:
Is there a reason we're requiring an API key to make this request? The endpoint `https://api.passage.id/v1/apps/{appID}` doesn't require authentication / an authorization header